vertica_python/vertica/connection.py

```diff
@@ -454,6 +454,7 @@ def _socket(self):
             raw_socket = self.enable_ssl(raw_socket, ssl_options)
 
         self.socket = raw_socket
+        self.socket_as_file = self.socket.makefile('rb', 10*1024*1024)
```

Comment:
How will the buffer size affect the performance? Is it possible the size is too large? What happened if you don't specify a value and use the default?

Reply:
Thanks for the answer. I prefer to use the default buffer size.
`self.socket_as_file = self.socket.makefile('rb')`

Reply:
Done.

Just making sure we are aligned, the default is `None`.